### DISCUSSION TITLE
Question about ASSIMP_INJECT_DEBUG_POSTFIX

### DISCUSSION THREAD

**Discussion by mickeyordog** [assimp/assimp#5917] [Q&A] [answered]
Would someone be able to provide more information about what this means? It's an option in the CMakeLists.txt. I'm wondering if this is something I want to enable/disable based on whether my project is built in debug or release

**Comment by kimkulling** [comment, marked as answer]
This is an option if you need to add a special postfix into the build artifacts. So for instance if you need a postfix like:

`libassimpd.so (unix) / assimpd.dll (windows)` this option helps you to define it.
